Gems/Terrain/Code/Source/TerrainSystem/TerrainSystem.h

```diff
@@ -65,6 +65,7 @@ namespace Terrain
         AZ::Aabb GetTerrainAabb() const override;
         void SetTerrainAabb(const AZ::Aabb& worldBounds) override;
 
+        bool TerrainAreaExistsInBounds(const AZ::Aabb& bounds) override;
```

Comment:
Shouldn't this also appear in TerrainDataRequestBus, and have a mock created for it? I'm not sure what it's overriding right now.

Reply:
Yeah totally missed committing these files since they're in a different folder location.